SPECS/python-prettytable/python-prettytable.spec

```diff
@@ -1,16 +1,20 @@
 Summary:        Library for displaying tabular data in a visually appealing ASCII format
 Name:           python-prettytable
-Version:        3.2.0
-Release:        2%{?dist}
+Version:        3.10.0
+Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          Development/Languages/Python
 URL:            https://github.com/jazzband/prettytable
 Source0:        https://pypi.io/packages/source/p/prettytable/prettytable-%{version}.tar.gz
+Patch0:         0001-replace-to-flit.patch
```

Comment:
can you add a comment for why this patch was needed

Reply:
the comment has been added to the changelog